Tests/scripts/validate_files_structure.py

```diff
@@ -177,13 +176,30 @@ def validate_schema(file_path, matching_regex=None):
 
 def changed_id(file_path):
     change_string = run_git_command("git diff HEAD {0}".format(file_path))
-    if re.search("\+id: .*", change_string) or re.search("\-id: .*", change_string):
-        print_error("You've changed the ID of the playbook {0} please undo.".format(file_path))
+    if re.search("[+-]( )?id: .*", change_string):
```

Comment:
in the original regex there was double space in between the +/- and the word id.
git removed the duplication in my comment (sorry about that).